doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1349 @@
+
+* The `previous method to define a board (hardware model)`_ is deprecated and being replaced by :ref:`a new method (hardware model v2) <zephyr:board_porting_guide>`.
+* The previous method to define :ref:`multi-image builds (parent-child images) <ug_multi_image>` is deprecated and being replaced by :ref:`zephyr:sysbuild`.
+
+All samples and applications in the SDK have been migrated.
+Consult respective documentation as additional or changed parameters might be needed to build them successfully.
+Applications that are outside of the SDK and use custom-defined boards should not be affected.
+nRF Connect for VS Code users migrating to the latest version of the SDK might be affected.
+
+All samples and applications in the SDK are built with sysbuild by default.
+Applications that are outside the SDK are not built with sysbuild by default.
+
+The deprecated methods are scheduled for removal after the next release.
+We recommend transitioning to the alternatives as soon as possible.
+Consult migration guides for `Migrating to the current hardware model`_ and `Migrating from multi-image builds to sysbuild`_.
+Exercise caution when migrating production environments to the latest SDK.
+
+Added the following features as supported:
+
+* Matter:
+
+  * Matter 1.3, which enables energy reporting for Matter devices and support for water and energy management, electric vehicle charges, and new major appliances.
+    Learn more about Matter 1.3 in the `Matter 1.3 CSA blog post`_.
+
+* Wi-Fi®:
+
+  * Wi-Fi and Thread coexistence.
+  * Software-enabled Access Point (SoftAP) for provisioning that was introduced in |NCS| v2.6.0 as experimental is now supported.
+  * Raw Wi-Fi reception in both Monitor and Promiscuous modes that was introduced in |NCS| v2.6.0 as experimental is now supported.
+  * New samples: :ref:`softap_wifi_provision_sample`, :ref:`wifi_promiscuous_sample`, and :ref:`wifi_thread_coex_sample`.
+
+* Other:
+
+  * Google Find My Device that allows the creation of a locator device that is compatible with the Android `Find My Device app`_.
+    This feature is showcased in the :ref:`fast_pair_locator_tag` sample.
+  * Hardware model v2 (HWMv2), an improved extensible system for defining boards.
+    This is the default boards definition system from this |NCS| release and onwards.
+    See `Migrating to the current hardware model`_.
+  * :ref:`zephyr:sysbuild`, an improved and extensible system for multi-image build, replacing :ref:`ug_multi_image` (parent/child images).
+    See `Migrating from multi-image builds to sysbuild`_.
+  * Samples and applications that use short-range radio and run on multi-core SoCs were migrated to use the :ref:`ipc_radio` as the default image for the network/radio core.
+    Samples previously used for the network/radio core are not supported any more: :ref:`zephyr:bluetooth-hci-ipc-sample`, :ref:`zephyr:nrf-ieee802154-rpmsg-sample`, :ref:`multiprotocol-rpmsg-sample`, and :ref:`ble_rpc_host`.
```

Comment:
```suggestion
    Samples previously used for the network/radio core are no longer used in the default builds: :ref:`zephyr:bluetooth-hci-ipc-sample`, :ref:`zephyr:nrf-ieee802154-rpmsg-sample`, :ref:`multiprotocol-rpmsg-sample`, and :ref:`ble_rpc_host`.
```

Reply:
Please modify as suggested. The samples are not deprecated yet and can be used in out-of-tree builds.